src/lv_draw/lv_draw_label.c

```diff
@@ -122,7 +122,11 @@ LV_ATTRIBUTE_FAST_MEM void lv_draw_label(const lv_area_t * coords, const lv_area
     int32_t w;
 
     /*No need to waste processor time if string is empty*/
-    if(txt[0] == '\0')  return;
+    if (txt != NULL) {
+        if(txt[0] == '\0')  return;
+    } else {
+        return;
+    }
```

Comment:
Small change to make it a bit easier to read.

```suggestion
    if (txt == NULL || txt[0] == '\0')
        return;
```